cmd/main.go

```diff
@@ -28,8 +36,15 @@ func main() {
 	app.Usage = "elements node + utxos scanner"
 	app.Commands = []*cli.Command{
 		{
-			Name:   "start",
-			Usage:  "run neutrino-elements node",
+			Name:  "start",
+			Usage: "watch for an address using utxo scanner",
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:    "address",
+					Usage:   "address to watch",
+					Aliases: []string{"a"},
+				},
+			},
```

Comment:
what about also a flag for the P2P host to connect to (we can tweak "localhost:18886" then)? Most likely should be a strings slice

```sh
neutrino-elements --connect "localhost:18886" --connect "localhost:28886" start --address <xxx>
```